pkg/didcomm/packer/jwe/authcrypt/unpack.go

```diff
@@ -22,22 +22,22 @@ import (
 // encrypted CEK.
 // The current recipient is the one with the sender's encrypted key that successfully
 // decrypts with recipientKeyPair.Priv Key.
-func (p *Packer) Unpack(envelope []byte) ([]byte, []byte, error) {
+func (p *Packer) Unpack(envelope []byte) ([]byte, []byte, []byte, error) {
```

Comment:
there were already too many return values. perhaps a struct would be in order?

Reply:
Also solves dogsledding in tests.

Reply:
Struct now used - the same Envelope struct that the packager returns is now created by the packer.